transform/mattermost-analytics/models/marts/web_app/platform/_platform__models.yml

```diff
@@ -11,8 +11,11 @@ models:
       - name: context_page_referrer          
       - name: channel                        
       - name: context_app_namespace          
-      - name: id                             
-      - name: user_actual_id                 
+      - name: id
```

Comment:
wonder if we can order/group these columns somehow (either alphabetically) or by importance (e.g. id first) or relationships (e.g. all context_* together, all date/time-related ones together, etc.)